examples/README.md

```diff
@@ -0,0 +1,19 @@
+# Examples
+
+These data and examples are built using MFEM and derived from their examples
+(https://mfem.org/examples/). libROM's examples are divided into two categories.
+
+## DMD
```

Comment:
Since this is an introductory README, it would be helpful to write out what DMD and PROM stand for, e.g.
`Dynamic mode decomposition (DMD)`